### PR TITLE
Refactor some of the things

### PR DESCRIPTION
**Pull request by codestreamtest**
foo

### CODE REVIEW

File: inventory.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ import { Container } from "../container";
 import { Logger } from "../logger";
 import { Foo } from "../foo";
 
+// this is on feature/fail
+// so that I can test moar PRs
+
```
Comment (planteater) [inline inventory.js:26]: Start test 1

**Review by planteater — COMMENTED** [review]
dun 1

File: inventory.js (hunk 1 above)
Comment (planteater) [reply inventory.js:26]: heya

**Comment by planteater** [comment]
new we single

```
    interface ArrayConstructor {
        // eslint-disable-next-line @typescript-eslint/array-type
        isArray(arg: ReadonlyArray<any> | any): arg is ReadonlyArray<any>;
    }
```
								
[inventory.js](https://github.com/codestreamtest/beerfridge/blob/59f506bfc303ef1200042303085bd1cc529a6396/inventory.js#L35-L38) (Lines 35-38)

File: inventory.js [hunk 2]
```diff
@@ -21,6 +21,9 @@ import { Container } from "../container";
 import { Logger } from "../logger";
 import { Foo } from "../foo";
 
+// this is on feature/fail
+// so that I can test moar PRs
```
Comment (codestreamtest) [inline inventory.js:25]: pr comment 1

Comment (planteater) [inline inventory.js:25]: pr comment check

**Pull request closed without merging**